airbyte-integrations/connectors/destination-databricks/src/main/kotlin/io/airbyte/integrations/destination/databricks/jdbc/DatabricksDestinationHandler.kt

```diff
@@ -295,4 +303,6 @@ class DatabricksDestinationHandler(
     ) {
         // do Nothing
     }
+
+    fun query(query: String): List<JsonNode> = jdbcDatabase.queryJsons(query)
```

Comment:
This method name could be more descriptive

Reply:
hm. how does `queryDestination(sql: String)` sound? anything more than that starts feeling more verbose than helpful

Reply:
Oh it's a verb. :)